steps/ionic-build/1.0.0/step.yml

```diff
@@ -0,0 +1,99 @@
+title: Ionic Build
+summary: Build ionic project for your platform. Examples on GitHub.
+description: |-
+  With this step you can build ionic apps easily.
+  Select your environment things and start building.
+
+  On our [GitHub page](https://github.com/bussmann-it/steps-ionic/blob/master/README.md) you can see complete Bitrise examples for iOS and Android.
+  We describe the whole process for the Ionic conference app. From creating certificate or keystore to build the app successfully.
+
+  If you have questions feel free to ask us in english and german language.
+  We will help you as soon as possible ;-)
+website: https://github.com/bussmann-it/steps-ionic
+source_code_url: https://github.com/bussmann-it/steps-ionic.git
+support_url: https://github.com/bussmann-it/steps-ionic/issues
+published_at: 2017-05-14T02:06:53.343409014+02:00
+source:
+  git: https://github.com/bussmann-it/steps-ionic.git
+  commit: d587cee45caaf377f5fb29c0c1590475f2f1a056
+host_os_tags:
+- osx-10.9
+- osx-10.10
+- osx-10.11
+- osx-10.12
+project_type_tags:
```

Comment:
Well, actually, you could simply remove `project_type_tags` or specify an empty list for it (`[]`). That means "it works with any kind of project type". Until we have a specific project type for ionic, it's easier to let this step show up for any project in the step lists (IMO)